beaker.gemspec

```diff
@@ -33,6 +32,9 @@ Gem::Specification.new do |s|
   # Run time dependencies
   s.add_runtime_dependency 'minitest', '~> 5.4'
   s.add_runtime_dependency 'minitar', '~> 0.6'
+  s.add_runtime_dependency 'pry-byebug', '~> 3.4.2'
```

Comment:
This needs to be runtime because the users of beaker will get it as a bundled gem ; and I don't know (and don't see obvious [documentation|http://bundler.io/v1.0/man/gemfile.5.html] for) whether bundler allows adding a gem with its dev dependencies.